modules/oe_translation_poetry/src/Poetry.php

```diff
@@ -21,11 +22,13 @@ class Poetry extends PoetryLibrary {
    *   The config factory.
    * @param \Drupal\Core\Logger\LoggerChannelFactoryInterface $loggerChannelFactory
    *   The logger channel factory.
+   * @param \Drupal\tmgmt\TranslatorManager $translatorManager
+   *   The translator plugin manager.
    */
-  public function __construct(ConfigFactoryInterface $configFactory, LoggerChannelFactoryInterface $loggerChannelFactory) {
-
+  public function __construct(ConfigFactoryInterface $configFactory, LoggerChannelFactoryInterface $loggerChannelFactory, TranslatorManager $translatorManager) {
+    $poetry_translator_definition = $translatorManager->getDefinition('poetry');
     $values = [
-      'identifier.code' => Settings::get('poetry.identifier.code'),
+      'identifier.code' => $poetry_translator_definition['default_settings']['identifier'],
```

Comment:
This is not correct. You are taking the hardcoded data from the the plugin definition. What happens if the config in the translator changes? It wont change the plkugin definition. I think we might not even need to set defaults in the plugin definition. Not sure why they are there but for sure this is not how they are used. Perhaps as the defalt value in the config form.